### ISSUE TITLE
Saving - Feature

### ISSUE THREAD

**Issue by MrTopom** [fyne-io/pixeledit#18]
Define the file format and define if zip it for the potential synch.

File should manage :
- Layers
- Brushes
- Palette

Format should be open and allow to add new features to come.

**Comment by andydotxyz** [comment]
This confused me a little - are you requesting we design a new file format, or support something more than simple bitmap file types?
I don't think the world needs more image file formats ;)

**Comment by MrTopom** [comment]
In fact the question is : Do you want to make an application to edit images OR an application to create project for sprite and animations ?

**Comment by andydotxyz** [comment]
Fair enough. This is currently a bitmap image editor. I don't know if it will be a sprite editor in the future, or if another app that uses a lot of this functionality would do that. It probably depends on what the community wants to do with this :)
